pkg/plugins/eventing/event.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2021 Nitric Pty Ltd.
+//
```

Comment:
Would recommend moving this type into the file: plugins.go

Reply:
go files are always package scoped, I'm curious as to what advantage moving it would provide?

Reply:
Idea is to develop some consistency around this with the various plugins. Do we put dependent type in with the services ore keep them in a separate file.